frontend/javascripts/oxalis/view/right-menu/tree_hierarchy_view.js

```diff
@@ -285,22 +286,22 @@ class TreeHierarchyView extends React.PureComponent<Props, State> {
     const hasSubgroup = anySatisfyDeep(node.children, child => child.type === TYPE_GROUP);
     const menu = (
       <Menu onClick={this.handleDropdownClick}>
-        <Menu.Item key="create" groupId={id}>
+        <Menu.Item key="create" data-group-id={id}>
```

Comment:
Fix for a React warning that I encountered a lot. React complained about the custom prop `groupId` that was not removed before adding the element to the DOM.